src/mongocrypt-ctx.c

```diff
@@ -76,21 +76,7 @@ bool mongocrypt_ctx_setopt_key_id(mongocrypt_ctx_t *ctx, mongocrypt_binary_t *ke
     if (!ctx) {
         return false;
     }
-
-    if (ctx->crypt->log.trace_enabled && key_id && key_id->data) {
-        char *key_id_val;
-        /* this should never happen, so assert rather than return false */
-        BSON_ASSERT(key_id->len <= INT_MAX);
-        key_id_val = _mongocrypt_new_string_from_bytes(key_id->data, (int)key_id->len);
-        _mongocrypt_log(&ctx->crypt->log,
-                        MONGOCRYPT_LOG_LEVEL_TRACE,
-                        "%s (%s=\"%s\")",
-                        BSON_FUNC,
-                        "key_id",
-                        key_id_val);
-        bson_free(key_id_val);
-    }
-
+    
```

Comment:
Whitespace appears to fail the [check-format|https://parsley.mongodb.com/evergreen/libmongocrypt_misc_check_format_patch_de39931fb3f6036dcbed04ea7dcc97c93b6ea100_68b9f25d4c91e300078154e0_25_09_04_20_11_10/0/task?bookmarks=0,1289] Evergeen task. Run the `format-all.sh` script to fix. I run it like:
```bash
MONGOCRYPT_PYTHON=/Users/kevin.albertson/.venv/bin/python ./etc/format-all.sh
```